### ISSUE TITLE
error: ‘isPlatformX11’ is not a member of ‘QX11Info’

### ISSUE THREAD

**Issue by 3vi1** [MaartenBaert/ssr#768]
When building clean using Qt version 5.12.8 on Ubuntu 20.04, compilation fails at:

```
[ 53%] Building CXX object src/CMakeFiles/simplescreenrecorder.dir/GUI/Icons.cpp.o
[ 54%] Building CXX object src/CMakeFiles/simplescreenrecorder.dir/GUI/MainWindow.cpp.o
/home/evil/src/ssr/src/GUI/MainWindow.cpp: In constructor ‘MainWindow::MainWindow()’:
/home/evil/src/ssr/src/GUI/MainWindow.cpp:76:16: error: ‘isPlatformX11’ is not a member of ‘QX11Info’
   76 |  if(!QX11Info::isPlatformX11()) {
      |                ^~~~~~~~~~~~~
make[2]: *** [src/CMakeFiles/simplescreenrecorder.dir/build.make:538: src/CMakeFiles/simplescreenrecorder.dir/GUI/MainWindow.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:164: src/CMakeFiles/simplescreenrecorder.dir/all] Error 2
make: *** [Makefile:130: all] Error 2

```

Is a required Qt library (qtx11extras?) missing from the linkage?  It's been a good while, but I built this on the same system with older versions of Qt with no issue.

**Comment by MaartenBaert** [comment]
The Qt version is probably too old. I should add another check for that.

**Comment by 3vi1** [comment]
Nevermind - I was running my old shortcut script that just did a plain cmake.   It built fine with "WITH_QT5=TRUE ./simple-build-and-install " once I saw you had added that script.  Thanks!

**Comment by big-tree-wang** [comment]
unsubscribe ------------------ 原始邮件 ------------------
发件人: "Jesse Litton"<notifications@github.com>
发送时间: 2020年4月28日(星期二) 凌晨2:28
收件人: "MaartenBaert/ssr"<ssr@noreply.github.com>;
抄送: "Subscribed"<subscribed@noreply.github.com>;
主题: Re: [MaartenBaert/ssr] error: ‘isPlatformX11’ is not a member of ‘QX11Info’ (#768)




 
Closed #768.
 
&mdash;
